lib/app/models/sequence_result.rb

```diff
@@ -9,7 +10,7 @@ class SequenceResult
       include DataMapper::Resource
       property :id, String, key: true, default: proc { SecureRandom.uuid }
       property :name, String
-      property :result, String
+      property :result, String, default: ResultStatuses::PASS
```

Comment:
I could see defaulting to pass causing some some issues to silently slip through.

Reply:
I talked to Rob about this, what we were doing before this was explicitly setting the result to PASS on before calling update_result_counts (see line 124 in sequence_base), which which was the only thing causing update_result_counts to work at all (because otherwise it was returning nil). Long story short, we decided to set the default value to PASS for now, and there is a ticket in JIRA to make a PENDING status work

Reply:
For now, removed default value and instead the update_result_counts will set the result to PASS if it is nil after looping through all the test cases

Reply:
This looks ok for now, but we need to address how we set results to pass everywhere more comprehensively in a future PR.  This will be especially important if we add more statuses like `pending.